examples/workflow/2_generate_data.ipynb

```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "Download this notebook :download:`here <2_generate_data.ipynb>`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 2: Generate data samples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this section, we will use the amplitude model that we created with the expert system to generate a data sample via hit & miss Monte Carlo. This requires us to work with the User Interface to ComPWA, the C++ back-end of `pycompwa`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.1 Generate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the previous step, we created an amplitude module and exported it as an XML file. We now need to go through a few steps to generate a phase space sample and a data sample for the decay $J/\\psi \\rightarrow \\gamma\\pi^0\\pi^0$:\n",
+    "\n",
+    "1. Import the section of particle definitions that is embedded in the XML file.\n",
+    "\n",
+    "2. Build a ``Kinematics`` object following the specifications of the dynamics in the XML model file (in this case, the helicity formalism). The second argument of this function is a particle list.\n",
+    "\n",
+    "3. Use this ``Kinematics`` object to generate a phase space sample as well. This sample defines the space on which to evaluate the intensities of the amplitude model.\n",
+    "\n",
+    "4. Build an ``Intensity`` object from the amplitude model. Because we want the intensities to be normalized, you have to pass the intensity builder function a phase space sample as an argument. Note that during the creation of the ``Intensity`` object, the ``Kinematics`` instance is updated with the subsystems required for the decay topology.\n",
```

Comment:
Link to the Intensity class of the python interface

Reply:
Links to the API cannot be done in a Markdown cell unless through URL (and that will fail because those pages don't exist yet). See the [note a bit lower](https://github.com/ComPWA/pycompwa/pull/74/files/e8748a60583543d3758bd941ce4962fd17d7ce42#diff-c6eb90c41c60db32c961176e2dcccc13R90).

Reply:
Oh I overlooked that